src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1357,7 +1346,7 @@ void PosixSignals::signal_sets_init() {
   #if defined(PPC64) || defined(AIX)
     sigaddset(&unblocked_sigs, SIGTRAP);
   #endif
-  sigaddset(&unblocked_sigs, SR_signum);
+  sigaddset(&unblocked_sigs, PosixSignals::SR_signum);
```

Comment:
Shouldn't need the PosixSignals::prefix in this method

Reply:
Here we are going from:
`void PosixSignals::signal_sets_init()`
to
` void signal_sets_init()`
so in this case we do need it.